ipld/read_test.go

```diff
@@ -365,6 +371,43 @@ func TestGetLeavesByNamespace_MultipleRowsContainingSameNamespaceId(t *testing.T
 
 }
 
+func TestBatchSize(t *testing.T) {
+	tests := []struct {
+		name      string
+		origWidth int
+	}{
+		{"2", 2},
+		{"4", 4},
+		{"8", 8},
+		{"16", 16},
+		{"32", 32},
+		// {"64", 64}, // test case too large for CI with race detector
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			ctx, cancel := context.WithTimeout(context.Background(), time.Second*time.Duration(tt.origWidth))
```

Comment:
32x32 ODS needs roughly 4 seconds
63x64 over 30s

Reply:
On ci, I guess, not locally

Reply:
So are you done with it?

Reply:
yes, I think it's ok as is